components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/js/idp_mgt_edit.js

```diff
@@ -667,7 +667,7 @@ function showHidePassword(element, inputId) {
 
 function emailValidator(name) {
     var errorMsg = "";
-    var emailPattern = /^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,4}$/;
+ var emailPattern = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
Can you fix the formatting, please?

Reply:
@vihanga-liyanage Better the line remains the same as it is a regex. What like 120 character is for readability. But for regex it would be easier to read with single line.

Reply:
I was talking about indentation.